### DISCUSSION TITLE
求各位大佬推荐一点动漫

### DISCUSSION THREAD

**Discussion by airforce81192** [share121/inter-knot#2633] [其他]
我个人比较喜欢青春猪头系列，缘*空，长瀞这些，不知道有没有类似的推荐

**Comment by Invincible-Fairy** [comment]
这边推荐个高木同学、古见同学

**Comment by Eurus1997** [comment]
四月是你的谎言，中二病也要谈恋爱，我想要吃掉你的胰脏。CLANNED。我好久不看番了，这些都是以前看的，都还不错<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >

**Comment by shiliuyexiafeng** [comment]
我心里危险的东西，不知道你看过没有<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 00zzxx** [comment]
这边推荐两个当季新番吧就：败犬女主太多了怎么办，魔法少女与恶曾是敌人，起码目前感觉都还不错的样子

**Reply by LuAn-afl** [reply]
魔法少女哪个不胃疼吗？

**Reply by 00zzxx** [reply]
> 魔法少女哪个不胃疼吗？

目前看还是很甜很治愈的。如果要说结局的话，op里有句歌词唱的好：保持尚未完结就好，这份感情无须谢幕。

**Comment by icenow188** [comment]
憧憬成为魔法少女<img src="https://bbs-static.miyoushe.com/static/2023/07/25/0302be52d1922a94378bd529cefbda6e_1187257128792751951.png" alt="诺登斯_盯" width="75" >
